Modules/socketmodule.c

```diff
@@ -5103,14 +5115,25 @@ static int sock_cloexec_works = -1;
 #endif
 
 /*ARGSUSED*/
+
+/*[clinic input]
+_socket.socket.__init__
+    family: int = -1
+    type: int = -1
+    proto: int = -1
+    fileno as fdobj: object = NULL
+[clinic start generated code]*/
+
 static int
-sock_initobj(PyObject *self, PyObject *args, PyObject *kwds)
+_socket_socket___init___impl(PySocketSockObject *self, int family, int type,
+                              int proto, PyObject *fdobj)
+ /*[clinic end generated code: output=8d20ec7eb58df701 input=c0966eb4587c4634]*/
 {
-    PySocketSockObject *s = (PySocketSockObject *)self;
-    PyObject *fdobj = NULL;
+    //PySocketSockObject *s = (PySocketSockObject *)self;
+    //PyObject *fdobj = NULL;
```

Comment:
Please remove all dead code. Applies to the whole PR.

Reply:
okay will do.